client/src/pages/Login/Login.tsx

```diff
@@ -41,18 +42,26 @@ export default function Login(): JSX.Element {
       <CssBaseline />
       <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
+          {/* --Commented out until thew new authHeader is completed
+          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />*/}
```

Comment:
If this is related to the routing issue, same applies here. Just make sure the `AuthHeader` is using the react-router-dom Link component.